grpc-example/grpc-example-java/README.md

```diff
@@ -1,107 +1,22 @@
 # Lagom gRPC Example (Java)
 
-This project demonstrates the usage of [akka-grpc](https://github.com/akka/akka-grpc) into Lagom.
+## Guide
```

Comment:
This gRPC guide was originally a `README` and was then migrated to paradox to publish on techhub. When migrating, this README was left over. In the scala example, though, the README was cleaned.